assets/components/productPage/productPageTabs/productPageTabs.jsx

```diff
@@ -27,13 +29,23 @@ const ProductPageTabs = ({
 }: PropTypes) => (
   <nav className="component-product-page-tabs">
     <ul className="component-product-page-tabs__ul">
-      {tabs.map(({ name }, index) => {
+      {tabs.map(({ name, href }, index) => {
         const isActive = index === active;
         return (
           <li className="component-product-page-tabs__li">
-            <button className={classNameWithModifiers('component-product-page-tabs__tab', isActive ? ['active'] : [])} onClick={() => onChange(index)}>
-              {name}
-            </button>
+            {href ? (
+              <a href={href} className={classNameWithModifiers('component-product-page-tabs__tab', isActive ? ['active'] : [])} onClick={(ev) => { ev.preventDefault(); onChange(index); }}>
```

Comment:
What scenario would we have an `a` over a `button`?

Reply:
they look like tabs but they are links to two pages with two different canonical urls, since they have an href they make more sense as a link. I think normally Buttons are for actions rather than navigation but things like tabbars are a bit muddy (actual tabbars actually have their own set of aria roles)

This tabbar is a bit more complex on top of that because it's not really a tabbar it's navigation between two pages (there are more differences than similarities) dressed as a tabbar.

(Plus search engines can follow the links and index both pages!)